f2/src/components/checkbox/index.js

```diff
@@ -47,15 +79,18 @@ export const Checkbox = ({ label, isChecked, ...props }) => {
                 {props.children}
               </Text>
             </Flex>
-          </Box>
+
+            {isChecked && (
+              <ConditionalForm>{props.conditionalField}</ConditionalForm>
```

Comment:
We might want `props.conditionalField` in the &&, otherwise if there’s no conditional it’ll still make an empty `<ConditionalForm/>`

Reply:
Oh, I had this issue. Good call